libs/backend/src/diagnostics.ts

```diff
@@ -56,7 +56,7 @@ export function getMostRecentDiagnosticRecord(
         timestamp
       from diagnostics
       where type = ?
-      order by timestamp desc
+      order by id desc
```

Comment:
On all our machines, the diagnostics table ID is a monotonically increasing integer:

https://github.com/votingworks/vxsuite/blob/576efc196103513036c671c3149fc1822d9926e3/apps/scan/backend/schema.sql#L65